app/controllers/course/condition/achievements_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Course::Condition::AchievementsController < Course::ConditionsController
+  load_resource instance_name: :achievement_condition, class: Course::Condition::Achievement.name,
+                param_method: :achievement_condition_params
+
+  def new
+  end
+
+  def edit
+  end
+
+  def create
```

Comment:
It's just the matter of convention and I have no objection to either way. But our code needs to be consistent. Do you still want to change all controllers to follow this?